site/in-dev/configuration.md

```diff
@@ -189,20 +189,20 @@ Related Quarkus settings:
+| H2               | only for development and local testing           | `JDBC`                                                  | Do not use for any serious use case.                                                                                                                                                                                                                                                                      |
+| MariaDB          | experimental, feedback welcome                   | `JDBC`                                                  |                                                                                                                                                                                                                                                                                                           |
+| MySQL            | experimental, feedback welcome                   | `JDBC`                                                  | Works by connecting the MariaDB driver to a MySQL server.                                                                                                                                                                                                                                                 |
+| CockroachDB      | experimental, known issues                       | `JDBC`                                                  | Known to raise user-facing "write too old" errors under contention.                                                                                                                                                                                                                                       |
+| Apache Cassandra | experimental, known issues                       | `CASSANDRA`                                             | Known to raise user-facing errors due to Cassandra's concept of letting the driver timeout too early, or database timeouts.                                                                                                                                                                               |
+| ScyllaDB         | experimental, known issues                       | `CASSANDRA`                                             | Known to raise user-facing errors due to Cassandra's concept of letting the driver timeout too early, or database timeouts. Known to be slow in container based testing. Unclear how good Scylla's LWT implementation performs. With Scylla version 6, you need to use a keyspace with "tables" disabled. |
```

Comment:
tables -> tablets

Reply:
Oh, that's something different :facepalm: